viewer/src/models/cad/types.ts

```diff
@@ -106,6 +106,9 @@ export type InstancedMesh = {
 // TODO 2019-12-05 larsmoa: Rename to e.g. SectorGeometry to avoid
 // confusion with other Sector-class
 export interface Sector {
+  readonly nodeIdToTreeIndexMap: Map<number, number>;
+  readonly treeIndexToNodeIdMap: Map<number, number>;
```

Comment:
Should we store this per sector? Seems like this only is intermittent data, and _only_ (?) used by the migration wrapper? Is there any way we can avoid storing the data when we don't need it? I'm thinking the map should live somewhere inside `Cognite3dModel`, but I'm open to discuss this.

Reply:
Is the type you are commenting on here not also only intermittent? Where would you otherwise store it? I think we throw away this data once the sector is consumed.

Reply:
We could store the final map in Cognite3dModel and not the root sector, though. That probably makes sense.

Reply:
Do we need to have this intermittently stored in `Sector`? Gives me the creeps to have collections that are valid only for a brief period...

Reply:
I do not think I understand what you prefer instead? How can we avoid storing the maps in a data structure after parsing the files? All the data in `Sector` only exists for a limited amount of time.

Reply:
Closing this as we discussed it offline :)